src/browser/modules/Sidebar/Sidebar.tsx

```diff
@@ -79,6 +79,7 @@ const Sidebar = ({
 }: SidebarProps) => {
   const topNavItemsList: NavItem[] = [
     {
+      // Consider use constant variable to store those keys
```

Comment:
What exactly do you mean here? Store the names and titles?

Reply:
Eija has suggested we start using this lib: https://www.npmjs.com/package/i18n to store all our user facing texts as variables

Reply:
I think it'd be nice to use something like i18n to store all our user facing text as variables 🤔

Reply:
@OskarDamkjaer - I'm not sure if we need to hard-code the item name as the key in any cases (I haven't found any hard-code so far). But if we need in any cases, IMO it's better to use some variables like:

```ts
const DOCUMENTS = 'documents'
```

```ts
const bottomNavItemsList: NavItem[] = [
  {
    name: DOCUMENTS,
    // rest of the code
  }
]
```

But if there's not any cases to get access to those keys manually, we can just leave it as it is.

Reply:
The guide navigation item already exists as a variable "GUIDE_DRAWER_ID" but it's not used in this file yet. I think it'd make sense to use that variable and add the others as well. Then we could also update the `NavItem` type to also use these types/variables.

Reply:
It'd be nice to have done, but not really needed for this PR unless you really want to do it

Reply:
@OskarDamkjaer - I'd prefer to leave it in the future!